src/Microsoft.DotNet.ProjectJsonMigration/ProjectContextExtensions.cs

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using Microsoft.Build.Construction;
+using System.Linq;
+using Microsoft.DotNet.ProjectModel;
+using System.IO;
+
+namespace Microsoft.DotNet.ProjectJsonMigration
+{
+    public static class ProjectContextExtensions
+    {
+        public static string GetProjectName(this ProjectContext projectContext)
+        {
+            return Path.GetFileNameWithoutExtension(projectContext.GetOutputPaths("_").CompilationFiles.Assembly);
```

Comment:
`_` is a common pattern in functional languages for something that you don't care about.

I can put a comment in there though.